src/DetailsView/components/command-bar-buttons-menu.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { CommandBarProps } from 'DetailsView/components/details-view-command-bar';
+import { CommandBarButton, IOverflowSetItemProps, OverflowSet } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+export type CommandBarButtonsMenuProps = CommandBarProps;
+
+export const CommandBarButtonsMenu = NamedFC<CommandBarButtonsMenuProps>(
+    'DetailsListIssuesView',
+    props => {
+        const onRenderItem = (item: IOverflowSetItemProps) => {
```

Comment:
So ... onRenderItem doesn't return anything? A bit confused about how this works ...

Reply:
Not sure how it was working before, but I added the return statement

Reply:
Should we also include the function return type here, as in onRenderOverflowButton?